src/features/settings/settingsSlice.ts

```diff
@@ -16,18 +17,48 @@ export interface SettingsState {
   explorerLiquidAPI: string;
   explorerLiquidUI: string;
   assets: Asset[];
+  baseUrl: string;
+  useProxy: boolean;
   marketsLabelled?: MarketLabelled[];
-  tdexDaemonBaseUrl: string;
   macaroonCredentials?: string;
   tdexdConnectUrl?: string;
+  proxyCommand?: Child;
 }
 
+export const startProxy = createAsyncThunk<Child, void, { state: RootState }>(
+  'settings/startProxy',
+  async (_, thunkAPI) => {
+    const { settings } = thunkAPI.getState();
+    console.log('HEYHEY', settings);
+    if (settings.proxyCommand) {
+      console.log('stopping proxy');
+      settings.proxyCommand.kill();
+    }
+    console.log('starting up proxy with url', settings.tdexdConnectUrl);
+    const cmd = await Command.sidecar('grpcproxy', ['--tdexdconnecturl', settings.tdexdConnectUrl!]).spawn();
+    console.log('done!!!', cmd.pid);
```

Comment:
```suggestion
    console.log('rpc proxy is running', cmd.pid);
```

Reply:
I forgot to remove these logs. I'll completely remove them.